fe/src/pages/Home/Home.tsx

```diff
@@ -1,29 +1,14 @@
-import {
-  Box, Code, Image, Text,
-} from '@chakra-ui/react';
-import { Link } from 'react-router-dom';
+import { Flex, Image } from '@chakra-ui/react';
 import logo from '../../assets/logo.svg';
 import AppLink from './AppLink';
 import styles from './Home.module.scss';
 
 function Home(): JSX.Element {
   return (
-    <Box className={styles.home}>
-      <Box className={styles.homeHeader}>
-        <Image src={logo} className={styles.homeLogo} alt="logo" />
-        <Text>
-          Edit
-          {' '}
-          <Code>src/Home.tsx</Code>
-          {' '}
-          and save to reload.
-        </Text>
-        <Text>
-          <Link to="/about">About</Link>
-        </Text>
-        <AppLink />
-      </Box>
-    </Box>
+    <Flex w="100%" minH="100vh" justifyContent="center" flexDirection="column" alignItems="center">
```

Comment:
Same goes here with comment above. ^

Reply:
Doesn't matter how page content is aligned atm. Could you try to use a Box element there instead?

Reply:
Actually you can let it be, with near upcoming changes it will be changed anyway.